### PR TITLE
improve viewing diagnostics article

### PR DESCRIPTION
**Pull request by Oblarg**
Adds section for common errors and their diagnostic plots.

### CODE REVIEW

**Comment by sciencewhiz** [comment]
highthreshold-voltage.PNG should have lowercase extension

**Comment by AustinShalit** [comment]
This PR has references to proposed documentation in #512. We should wait on this PR until #512 is resolved.

**Comment by Oblarg** [comment]
Wording has been updated to reflect root-cause uncertainty.

File: source/docs/software/wpilib-tools/robot-characterization/viewing-diagnostics.rst [hunk 1]
```diff
@@ -63,3 +51,51 @@ This plot is interactive, and may be rotated by clicking-and-dragging. The quasi
...
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: images/highthreshold-voltage.png
+   :alt: Voltage domain plot with threshold too high
+
+While not nearly as problematic as a too-low threshold, a motion threshold that is too high will result in a large "gap" in the voltage domain quasistatic plot.
+
+To solve this, decrease the setting and re-analyze the data.
+
+Magnetic Encoders Near Motors
```
Comment (sciencewhiz) [inline source/docs/software/wpilib-tools/robot-characterization/viewing-diagnostics.rst]:
```suggestion
Magnetic Encoders Velocity Noise
```

**Pull request merged**